### PR TITLE
update exe name to readme and put more instructions under linux

### PR DESCRIPTION
**Pull request by immartian**
nothing special but some clearer instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
 The official Musicoin Wallet
 
-# Getting Started
+# Running on Ubuntu (for Window/Mac, just download, unzip and run up Musicoin-wallet)
+
+Step 1: Download wallet:
+https://github.com/Musicoin/desktop/releases (find the latest version for Linux)
+Step 2: Unzip the tar file
+Step 3: Open terminal in the root folder of the MusiCoin folder
+Step 4: type ./Musicoin-wallet and hit enter
```
Comment (KumaraDev) [inline README.md]: Step 3.a type sudo linux-setup.sh in the root folder. The bash file will install libnss3 and gtk2.0, then chmod Musicoin-wallet and gmc.

Comment (GnomesNeedLoveToo) [inline README.md]: Type 'sudo linux-setup.sh' in the root folder. The bash file will install libnss3 and gtk2.0, then chmod Musicoin-wallet and gmc.

Comment (Varunram) [reply README.md]: Lets add that under requirements?

Comment (Varunram) [reply README.md]: Done.

**Pull request merged**